react/components/molecules/radio-group/radio-group.js

```diff
@@ -0,0 +1,113 @@
+
+import { Radio } from "../../atoms";
+
+export class RadioGroup extends mix(PureComponent).with(IdentifiableMixin) {
+    static get propTypes() {
+        return {
+            items: PropTypes.arrayOf(
+                PropTypes.exact({
+                    label: PropTypes.string,
+                    value: PropTypes.string.isRequired,
+                    disabled: PropTypes.bool,
+                    error: PropTypes.bool
+                })
+            ),
+            values: PropTypes.object,
+            error: PropTypes.bool,
+            disabled: PropTypes.bool,
+            beforeItem: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
+            afterItem: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
+            onUpdateValue: PropTypes.func,
+            style: ViewPropTypes.style
+        };
+    }
+
+    static get defaultProps() {
+        return {
+            items: [],
+            values: {},
+            error: false,
+            disabled: false,
+            beforeItem: undefined,
+            afterItem: undefined,
+            onUpdateValue: () => {},
+            style: {}
+        };
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            valueData: props.value
+        };
+    }
+
+    componentDidUpdate(prevProps) {
+        if (prevProps.value === this.props.value) return;
+        this.setState({
+            valueData: this.props.value
+        });
+    }
+
+    onUpdateChecked = value => {
+        this.setState(
+            {
+                valueData: value
+            },
+            () => this.props.onUpdateValue(value)
+        );
+    };
+
+    _renderRadios = () => {
+        return this.props.items.map((item, index) => (
+            <View style={styles.radioItem} key={item.value}>
+                {this.props.beforeItem &&
+                    React.cloneElement(this.props.beforeItem, {
+                        index: index,
+                        item: item,
+                        checked: item.value === this.state.valueData
+                    })}
+                <Radio
+                    label={item.label || item.value}
+                    value={item.value}
+                    checked={item.value === this.state.valueData}
+                    disabled={item.disabled || this.props.disabled}
+                    variant={item.variant || this.props.error ? "error" : null}
+                    onUpdateChecked={this.onUpdateChecked}
+                />
+                {this.props.afterItem &&
+                    React.cloneElement(this.props.afterItem, {
+                        index: index,
+                        item: item,
+                        checked: item.value === this.state.valueData
+                    })}
+            </View>
+        ));
+    };
+
+    render() {
+        return (
+            <View style={[styles.radioGroup, this.props.style]} {...this.id("radio-group")}>

```

Comment:
Even tho there's no conditionals in setting style, maybe move all style to a `_style()` const still